src/opencost_parquet_exporter.py

```diff
@@ -271,22 +272,34 @@ def save_result(processed_result, config):
         print("Error: Incomplete AWS credentials provided for accessing S3")
     except boto_exceptions.ClientError as ce:
         print(f"AWS Client Error: {ce}")
+    return None
 
 def main():
     """
     Main function to execute the workflow of fetching, processing, and saving data
     for yesterday.
     """
+    print("Starting run for yesterday data")
```

Comment:
I feel like this is a good default message, it may make sense for it to be "Starting run". If you pass in a specific time frame (like todays data for instance), that might be confusing to end users. I don't think this is critical, just nice to have.